canarieapi/app_object.py

```diff
@@ -10,7 +10,7 @@
 from os import environ
 
 # -- Project specific --------------------------------------------------------
-import default_configuration
+from . import default_configuration
```

Comment:
In the past I found that doing this was breaking a lot depending on how the environment is created or where `.` refers to (workdir, another local file with same name, some other package with same name found before, etc.).
The safest is to do `from canarieapi import default_configuration`.
If the package is installed, this guarantees to find the right thing.

Reply:
Actually, scratch that. 
I just remembered that the explicit reason why this one is imported like so instead of `from canarieapi ...` is to allow override on the **local** configurations by the docker image. Using `from ...` will ignore the local file and use the installed one in site-packages.
The calling process should then make sure to add the local path with `sys.insert(0, SOURCE)`.

Reply:
I too was not so sure about the "from .". 
So you mean that we should keep the  "import default_configuration" line as is but adding the sys.insert(0, /opt/local/src/CanarieAPI/canarieapi) in the cron command?

Reply:
hmm... I was actually taught (and used in the past) `from module import abc` to import from local modules. the `from . ` , I found while looking for the ModuleNotFound error

Reply:
If the docker did a pip install of canarieapi, `from canarieapi` will refer to the site-packages, but in pavics-compose, this config is overridden at the source location. So we end up overriding a file that doesn't get used. So yes, like @dbyrns said.

Reply:
Note that in the eyes of the imported module that will be running, 'local' means site-packages if it was installed. The dot is a relative to self.

Reply:
I modified the cron command. Added a comment for the 'why' `default_configuration` is imported that way and how to manage the calling process